SunEngine.Commons/Configuration/AddServices/AddStores.cs

```diff
@@ -21,5 +27,28 @@ public static void AddStores(this IServiceCollection services, DataBaseFactory d
 
             services.AddSingleton<SpamProtectionCache>();
         }
+
+        // Temporary solution
+        public static void AddCachePolicy(this IServiceCollection services)
+        {
+            services.AddScoped<ICachePolicy>(provider =>
+            {
+                var cacheOptions = provider.GetRequiredService<IOptions<CacheOptions>>();
```

Comment:
I would suggest extracting this to the method - "getCahcePolicy" to separate abstraction levels better.